src/docs/blueprint_documentation.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Packaging Theia as a Desktop Product
+---
+
+# Packaging Theia as a Desktop Product
+
+Eclipse Theia Blueprint product is a template to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform.
```

Comment:
I believe it can be reworked to be more clear, and read better:

```suggestion
Eclipse Theia Blueprint is an example product used as a reference on how to build desktop IDE-like products based on the Eclipse Theia framework.
```